PyDSS/cli/run.py

```diff
@@ -68,7 +68,6 @@
 def run(project_path, options=None, tar_project=False, zip_project=False, verbose=False, simulations_file=None, dry_run=False):
     """Run a PyDSS simulation."""
     if not os.path.exists(project_path):
-        print(f"project-path={project_path} does not exist")
```

Comment:
Did you intend to remove these print statements? They seem helpful.

Reply:
added